js/src/tooltip.js

```diff
@@ -283,7 +283,11 @@ class Tooltip extends BaseComponent {
 
     EventHandler.trigger(this._element, this.constructor.Event.INSERTED)
```

Comment:
Also, please consider this event. Depending on how you anticipate this event to be used by people, it may make most sense to only trigger it on true, _new popper_ creation, otherwise it will be triggered every time someone re-enters a tooltip trigger while its transitioning to hidden.

Reply:
True, it should be triggered only when the tooltip is inserted in the DOM.